static/src/js/containers/MapContainer/MapWrapper.js

```diff
@@ -0,0 +1,276 @@
+            resolution="500m"
+            format="jpeg"
+          />
+        </LayersControl.BaseLayer>
+        <LayersControl.BaseLayer
+          checked={base.trueColor}
+          name="Corrected Reflectance (True Color)"
+        >
+          <LayerBuilder
+            projection={projection}
+            product="VIIRS_SNPP_CorrectedReflectance_TrueColor"
+            resolution="250m"
+            format="jpeg"
+            time
+          />
+        </LayersControl.BaseLayer>
+        <LayersControl.BaseLayer
+          checked={base.landWaterMap}
+          name="Land / Water Map *"
+        >
+          <LayerBuilder
+            projection={projection}
+            product="OSM_Land_Water_Map"
+            resolution="250m"
+            format="png"
+          />
+        </LayersControl.BaseLayer>
+        <LayersControl.Overlay
+          checked={overlays.referenceFeatures}
+          name="Borders and Roads *"
+        >
+          <LayerBuilder
+            projection={projection}
+            product="Reference_Features_15m"
+            resolution="15.625m"
+            format="png"
+          />
+        </LayersControl.Overlay>
+        <LayersControl.Overlay
+          checked={overlays.coastlines}
+          name="Coastlines"
+        >
+          <LayerBuilder
+            projection={projection}
+            product="Coastlines_15m"
+            resolution="15.625m"
+            format="png"
+          />
+        </LayersControl.Overlay>
+        <LayersControl.Overlay
+          checked={overlays.referenceLabels}
+          name="Place Labels *"
+        >
+          <LayerBuilder
+            projection={projection}
+            product="Reference_Labels_15m"
+            resolution="15.625m"
+            format="png"
+          />
+        </LayersControl.Overlay>
+      </LayersControl>
+      <ZoomHome
+        projection={projection}
+      />
+      {
+        !isProjectPage && (
+          <ProjectionSwitcher
+            position="bottomright"
+            onChangeProjection={onChangeProjection}
+          />
+        )
+      }
+      <ScaleControl position="bottomright" />
+      <ConnectedSpatialSelectionContainer mapProps={mapProps} />
+      <GranuleGridLayer
+        collectionsMetadata={collectionsMetadata}
+        drawingNewLayer={drawingNewLayer}
+        focusedCollectionId={focusedCollectionId}
+        focusedGranuleId={focusedGranuleId}
+        granules={granules}
+        granulesMetadata={granulesMetadata}
+        imageryCache={imageryCache}
+        isProjectPage={isProjectPage}
+        onChangeFocusedGranule={onChangeFocusedGranule}
+        onExcludeGranule={onExcludeGranule}
+        onMetricsMap={onMetricsMap}
+        project={project}
+        projection={projection}
+      />
+      <MouseEventsLayer />
+      <ShapefileLayer
+        authToken={authToken}
+        isProjectPage={isProjectPage}
+        shapefile={shapefile}
+        onChangeProjection={onChangeProjection}
```

Comment:
curious what you'd think about this syntax
```js
<ShapefileLayer { ...{ authToken, isProjectPage, shapefile, onChangeProjection } } />
```

Reply:
eslint doesn't like it when we spread props like that